tests/torch/sparsity/movement/test_training_with_third_party.py

```diff
@@ -1,87 +0,0 @@
-# Copyright (c) 2023 Intel Corporation
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#      http://www.apache.org/licenses/LICENSE-2.0
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-from pathlib import Path
-
-import pytest
-
-from tests.torch.helpers import Command
-from tests.torch.sparsity.movement.helpers import MRPC_CONFIG_FILE_NAME
-from tests.torch.sparsity.movement.helpers import TRAINING_SCRIPTS_PATH
-from tests.torch.test_sanity_third_party import TransformersVirtualEnvInstaller
-from tests.torch.test_sanity_third_party import create_command_line
-
-
-@pytest.fixture(scope="class")
-def temp_folder(tmp_path_factory):
-    root_folder = tmp_path_factory.mktemp("movement_third_party")
-    folders = {"models": root_folder / "models", "venv": root_folder / "venv", "repo": root_folder / "repo"}
-    for folder in folders.values():
-        Path(folder).mkdir(exist_ok=True, parents=True)
-    return folders
-
-
-@pytest.mark.usefixtures("temp_folder")
-class TestMovementWithTransformers:
-    @pytest.fixture(autouse=True)
-    def setup(self, temp_folder):
-        self.temp_folder = temp_folder
-        self.env = TransformersVirtualEnvInstaller(temp_folder["venv"], temp_folder["repo"])
-
-    @pytest.mark.dependency(name="install_transformers")
-    def test_install_transformers_env(self, third_party, pip_cache_dir):
-        if not third_party:
-            pytest.skip(
-                "Skip tests of movement sparsity with patched transformers package "
-                "since `--third-party-sanity` is False."
-            )
-        self.env.install_env(pip_cache_dir)
-
-    @pytest.mark.dependency(depends=["install_transformers"], name="glue_movement_train")
-    def test_movement_glue_train(self):
-        nncf_config = TRAINING_SCRIPTS_PATH / MRPC_CONFIG_FILE_NAME
-        output_dir = Path(self.temp_folder["models"], nncf_config.stem)
-        com_line = (
-            "examples/pytorch/text-classification/run_glue.py --model_name_or_path "
```

Comment:
There are similar tests in the Optimum:
https://github.com/huggingface/optimum-intel/blob/main/tests/openvino/test_training_examples.py#L87